File: C-CPP-Java-Logic-Building/11_Assignment/Assignment11_3.c
/* Programm to accept a number from user di andsplay pattern
    input = 5 
    output- 1 * 2 * 3 * 4 * 5 *
*/

#include<stdio.h>

int Pattern(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        printf("%d \t * \t",iCnt);   
    }

}

int main()
{
    int iValue = 0, iRet = 0;

    printf("Enter the number : ");
    scanf("%d", &iValue);

    Pattern(iValue);

    return 0;
}

File: Python-Logic-Building/11_Assignment/Assignment11_3.py

def Sum_of_digits(no):
    if no == 0:
        return 0
    else:
        return (no % 10) + Sum_of_digits(no // 10)

def main():
    num = int(input("Enter number : "))
    iRet = Sum_of_digits(num)
    print("sum of %d is :%d "% (num,iRet))

if __name__ == "__main__":
    main()